pkg/model/openstackmodel/firewall.go

```diff
@@ -442,7 +442,7 @@ func (b *FirewallModelBuilder) addProtokubeRules(c *fi.ModelBuilderContext, sgMa
 			Direction:    s(string(rules.DirIngress)),
 			Protocol:     s(string(rules.ProtocolTCP)),
 			EtherType:    s(string(rules.EtherType4)),
-			PortRangeMin: i(3998),
+			PortRangeMin: i(3994),
```

Comment:
If this is for etcd manager can we add a comment?  I've been trying to do this with self documenting code and the method names, however I think its probably fine here with a comment as its just taking some of the work out of protokube.  

Also not sure if there is a sane way of checking that we are using etcd manager and only opening the port in the event that the provider is not `Legacy`.